tests/integration/300.command.dev.test.js

```diff
@@ -258,5 +258,29 @@ testMatrix.forEach(({ args }) => {
       })
     })
   })
+  test(testName('should use .env vars instead of .env.development vars', args), async (t) => {
+    await withSiteBuilder('site-with-env-files', async (builder) => {
+      builder
+        .withNetlifyToml({
+          config: { dev: { envFiles: ['.env.development', '.env'] }, functions: { directory: 'functions' } },
```

Comment:
Based on the code, the the order from left to right, is low priority to high priority.
That's the reason the test is failing (it was only passing because of a bug in the test, see https://github.com/netlify/cli/pull/4321/commits/a239de2e0c3f6b3a86d08ad04d7b37cec2b939e9

Reply:
I see, you're right. In the test I purposedly inverted the order.

Reply:
This https://github.com/netlify/cli/pull/4321/commits/97380fa949a914ca7382e9821c8e723d48354b49 should fix it

Reply:
Finally. Yes it does